components/Form.js

```diff
@@ -36,12 +40,19 @@ export default function Form({
           ))}
         </select>
         <button type="submit">Submit</button>
-        <button type="button" onClick={handleFormReset}>
+        <button type="reset" onClick={handleFormReset}>
           Reset
         </button>
+        <button type="button" onClick={() => handleSurprise()}>
+          Surprise me
+        </button>
       </form>
 
       {formResults.length > 0 && <PreviewCard formResults={formResults} />}
+
+      {randomSurprise ? (
+        <SpotlightCard randomSurprise={randomSurprise} />
+      ) : null}
```

Comment:
by using the logical and here, you can make your code even more concise:

```suggestion
      {randomSurprise && <SpotlightCard randomSurprise={randomSurprise} />}
```